sentry_sdk/consts.py

```diff
@@ -539,6 +540,8 @@ def __init__(
         proxy_headers=None,  # type: Optional[Dict[str, str]]
         instrumenter=INSTRUMENTER.SENTRY,  # type: Optional[str]
         before_send_transaction=None,  # type: Optional[TransactionProcessor]
+        enable_sentry_logs=False,  # type: bool
```

Comment:
don't we also want to put these behind `_experiments`?

Reply:
good idea! yes, lets do this

Reply:
done.